app/test_main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+import pytest
+
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "url, connection, expected", [
+        (True, True, "Accessible"),
+        (True, False, "Not accessible"),
+        (False, True, "Not accessible"),
+        (False, False, "Not accessible"),
+    ]
+)
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_can_access_google_page(
+        mock_valid_url: callable,
+        mock_connect: callable,
+        url: bool,
+        connection: bool,
+        expected: str
+) -> None:
+    url_address = "www.google.com"
```

Comment:
There is no need to allocate the `url_address` variable, omit using it and place the value directly inside of the `can_access_google_page` function.